ArchiSteamFarm/Json/Steam.cs

```diff
@@ -390,9 +390,26 @@ internal class NumberResponse {
 
 		[SuppressMessage("ReSharper", "ClassCannotBeInstantiated")]
 		internal sealed class RedeemWalletResponse : EResultResponse {
+			internal sealed class Wallet {
+				[JsonProperty(PropertyName = "currency", Required = Required.Always)]
+				internal readonly ulong Amount;
+
+				[JsonProperty(PropertyName = "currencycode", Required = Required.Always)]
+				internal readonly ECurrencyCode CurrencyCode;
+
+				// Deserialized from JSON
+				private Wallet() { }
+			}
+
 			[JsonProperty(PropertyName = "detail", Required = Required.DisallowNull)]
 			internal readonly EPurchaseResultDetail? PurchaseResultDetail;
 
+			[JsonProperty(PropertyName = "currencycode", Required = Required.DisallowNull)]
+			internal readonly Wallet RedeemingCodeWallet;
+
+			[JsonProperty(PropertyName = "currency", Required = Required.DisallowNull)]
+			internal readonly ECurrencyCode? WalletCurrencyCode;
```

Comment:
`Wallet` here is redundant. This is already `RedeemWalletResponse`.

Reply:
It also conflicts with `Wallet.CurrencyCode` since it's not supposed to be the same. Maybe you meant `CodeCurrencyCode`?

Reply:
Or something to indicate what that is, `KeyCurrencyCode` also sounds good.

Reply:
no, it's account wallet code, so everything is ok

Reply:
What `Wallet.CurrencyCode` stands for then? One should be `KeyCurrencyCode` and other should be `AccountCurrencyCode`, right now I have no clue which is which.

Reply:
And it also doesn't help that one would be called `WalletCurrencyCode`, hence I suggest to explicitly name them key and account.